.env

```diff
@@ -1 +1,4 @@
 PORT=3001
+
+REACT_APP_BACKEND_HOST="http://localhost:3000"
+REACT_APP_BACKEND_VERSION="> 0.0.0-90"
```

Comment:
I think a good solution would be to specify the backend version in the `package.json` file (custom fields are allowed), but importing the file (and all other files outside of `src`) is currently not allowed. There is [a PR](https://github.com/facebookincubator/create-react-app/pull/2468) for that however that seems to be near completion.